Atomic/pull.py

```diff
@@ -27,6 +29,9 @@ def pull_docker_image(self):
         tag = tag if tag != "" else "latest"
         fq_name = skopeo_inspect("docker://{}".format(self.args.image))['Name']
         image = "docker-daemon:{}:{}".format(fq_name, tag)
+        if get_atomic_config_item(['discover_sigstores'], get_atomic_config()):
+            if not self.discover_sigstore():
```

Comment:
These should probably combined into a single function.  Put it in self.discover_sigstore()